gen/internals.py

```diff
@@ -284,6 +309,13 @@ class SkipError(Exception):
     pass
 
 
+# has a deterministic chunk before and after the variable name so we can deterministically get out
+# the name
+LATE_BIND_PLACEHOLDER_START = 'LATE_BIND_PLACEHOLDER_START_'
```

Comment:
Syntactically these are delimiters, and semantically (which we know because this isn't a general-purpose template language) they're the start and end of late bind placeholders.